integrations/komodor/models.py

```diff
@@ -0,0 +1,22 @@
+from enum import StrEnum
+
+from pydantic import BaseModel
+
+
+class IssueScope(BaseModel):
+    cluster: str
+
+
+class IssueProps(BaseModel):
+    type: str = "availability"
+    statuses: list[str] = ["open", "closed"]
+
+
+class IssueBody(BaseModel):
```

Comment:
sounds more like `IssueRequestBody` lets try to be specific to avoid confusions of different models in the future
```suggestion
class IssueRequestBody(BaseModel):
```